### PR TITLE
MOTECH-2170: Fixed showing ownerId in Commcare Cases

### PR DESCRIPTION
**Pull request by pkornowski**
Fixed showing ownerId in Commcare Cases, added test for this field. Previous PR I closed because of problem with git and mess on github, link: https://github.com/motech/modules/pull/137

### CODE REVIEW

File: commcare/src/main/java/org/motechproject/commcare/service/CommcareCaseService.java [hunk 1]
```diff
@@ -58,12 +58,33 @@
...
+     * Query CommCareHQ for all cases under a given user id.
+     *
+     * @param ownerId  the user id from CommCareHQ
+     * @param pageSize  the size of the page
+     * @param pageNumber  the number of the page
+     * @param configName  the name of the configuration used for connecting to CommcareHQ, null means default configuration
+     * @return  the list of CaseInfo objects representing cases under the given user id and page found on the given
+     *          CommcareHQ configuration
+     */
+
```
Comment (sebbrudzinski) [inline commcare/src/main/java/org/motechproject/commcare/service/CommcareCaseService.java]: What's with all those blank lines? Method signature should be just below the javadoc in the very next line

File: commcare/src/main/java/org/motechproject/commcare/service/CommcareCaseService.java [hunk 2]
```diff
@@ -58,12 +58,33 @@
...
      * Same as {@link #getCasesByUserId(String, Integer, Integer, String) getCasesByUserId} but uses default Commcare
      * configuration.
      */
     List<CaseInfo> getCasesByUserId(String userId, Integer pageSize, Integer pageNumber);
 
     /**
+     * Same as {@link #getCasesByOwnerId(String, Integer, Integer, String) getCasesByUserId} but uses default Commcare
+     * configuration.
+     */
+
```
Comment (sebbrudzinski) [inline commcare/src/main/java/org/motechproject/commcare/service/CommcareCaseService.java]: What's with all those blank lines? Method signature should be just below the javadoc in the very next line

File: commcare/src/main/java/org/motechproject/commcare/service/impl/CommcareCaseServiceImpl.java [hunk 3]
```diff
@@ -80,6 +80,16 @@ public CaseInfo getCaseByCaseId(String caseId, String configName) {
     }
 
     @Override
+    public List<CaseInfo> getCasesByOwnerId(String ownerId, Integer pageSize, Integer pageNumber, String configName) { //ONLY TEST
```
Comment (sebbrudzinski) [inline commcare/src/main/java/org/motechproject/commcare/service/impl/CommcareCaseServiceImpl.java]: Only test??

File: commcare/src/test/java/org/motechproject/commcare/service/impl/CommcareCaseServiceImplTest.java [hunk 4]
```diff
@@ -95,6 +95,22 @@ public void testCasesByUserId() {
...
+    public void testCasesByOwnerId() {
+        String ownerId = "testId";
+
+        CaseRequest request = new CaseRequest();
+        request.setOwnerId(ownerId);
+        request.setLimit(20);
+        request.setOffset(0);
+        when(commcareHttpClient.casesRequest(config.getAccountConfig(), request)).thenReturn(casesResponse());
+
+        List<CaseInfo> cases = caseService.getCasesByOwnerId(ownerId, 20, 1);
```
Comment (pgesek) [inline commcare/src/test/java/org/motechproject/commcare/service/impl/CommcareCaseServiceImplTest.java]: what I meant is to check the ownerId field in the testAllCases() test. So in other words, add a new assert to the existing test. We do not really need these new methods from what I understand.

**Pull request merged**